wwwapp/auth.py

```diff
@@ -26,21 +27,13 @@ def login_view(request):
         del request.session['merge_access_info']
 
     if request.user.is_authenticated:
-        try:
-            access = request.user.accountaccess_set.all()[0]
-        except IndexError:
-            access = None
-        else:
-            client = access.api_client
-            user_info = client.get_profile_info(raw_token=access.access_token)
-
-            user = request.user
-            try:
-                user_profile = UserProfile.objects.get(user=user)
-            except UserProfile.DoesNotExist:
-                new_user_info = UserInfo()
-                new_user_info.save()
-                user_profile, just_created = UserProfile.objects.get_or_create(user=user, user_info=new_user_info)
+        # This should never happen if the login flow worked correctly but I'm leaving this here just in case there are any broken users in the database already
+        # (aka I'm too afraid to remove it)
+        user_profile, just_created = UserProfile.objects.get_or_create(user=request.user)
+        if just_created:
+            logging.getLogger('django.request').error(
```

Comment:
I hope this works...

Reply:
Yes it does, I tested it by placing a breakpoint inside send_mail in Django

Reply:
OK

For easing your future testing: ```EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

Reply:
Hah I was looking for something like that :joy: